tools/downloader/topologies.py

```diff
@@ -0,0 +1,104 @@
+                in current directory and downloads to it. 
+        '''
+        if not download_dir: 
+            download_dir = Path('.').resolve() / 'models'
+        cache_dir = Path(download_dir) / '.cache'
+
+        parser = argparse.ArgumentParser()
+        args = argparse.Namespace(all=False, list=None, name=model_name, print_all=False)   
+        topology = common.load_models_from_args(parser, args)[0]
+
+        model_dir = download_dir / topology.subdirectory
+
+        flags = ['--name=' + model_name,
+                 '--output_dir=' + str(download_dir).replace('\\', '\\\\'), 
+                 '--cache_dir=' + str(cache_dir).replace('\\', '\\\\')]
+        if precisions:
+            flags += ['--precisions=' + ','.join([str(elem) for elem in precisions]) ]
+        
+        downloader.main(flags)
+
+        self.name = model_name
+        self.model_dir = model_dir
+        self.download_dir = download_dir
+        self.precision = precisions
+        self.precisions = topology.precisions
+        Model.__doc__ = Model.__doc__.format(topology.description, 
+                                                   topology.task_type, topology.license_url)
+
+    def get_ir(self, *, precision=None):
+        '''
+        Creates OpenVINO Intermediate Representation (IR) network model.
+        If the precision doesn't set, takes the precision value from the constructor 
+        or throws an error if it's also doesn't set.
+        If there is already converted model or the model has been downloaded in
+        IR - just returns the paths to the files.
+
+        Parameters
+        ----------
+            precision
+                Target model precision. By default is None.
+
+        Returns
+        -------
+            xml_path
+                Path to generated or downloaded .xml file
+            bin_path
+                Path to generated or downloaded .bin file
+        '''
+        if self.precision and len(self.precision) > 1 and (not precision or precision not in self.precision):
+            raise ValueError('Incorrect precision value!\n    Current precision value is: \''
+                                        + str(precision) +  '\'\n    Downloaded models precision values are: ' 
+                                        + str(self.precision))
+        
+        if self.precision and precision and precision != self.precision[0]:
+            raise ValueError('Downloaded ({}) and requested ({}) precisions do not match'.format(self.precision[0], precision))
+
+        res_precision = precision if precision else self.precision[0]
+        if res_precision not in self.precisions:
+            raise ValueError('Incorrect precision value!\n    Current precision value is: \'' 
+                                        + str(res_precision) + '\'.\n    Allowed precision values are: ' 
+                                        + str(self.precisions))
+  
+        prefix = Path(self.model_dir) / res_precision / self.name
```

Comment:
```suggestion
        prefix = self.model_dir / res_precision / self.name
```

`model_dir` is already a `Path`.